ledger/src/blockstore_processor.rs

```diff
@@ -1132,7 +863,10 @@ pub fn confirm_slot(
         recyclers,
         log_messages_bytes_limit,
         prioritization_fee_cache,
-    )
+        replayer_handle,
+    )?;
+
+    Ok(true)
```

Comment:
seems like this'd be better than returning `true` here:
```rust 
let slot_full = slot_entries_load_result.3;
confirm_slot_entries(...)?;
Ok(!slot_full)
```

that is, don't bother calling `confirm_slot()` again if the slot is full since there's nothing more to process by definition. `Ok(!bank.is_complete())` could be another way to express this.